wurst/war3map.j

```diff
@@ -0,0 +1,447 @@
+    call SetStartLocPrio(7, 0, 8, MAP_LOC_PRIO_HIGH)
+
+    call SetStartLocPrioCount(8, 2)
+    call SetStartLocPrio(8, 0, 7, MAP_LOC_PRIO_HIGH)
+    call SetStartLocPrio(8, 1, 9, MAP_LOC_PRIO_HIGH)
+
+    call SetStartLocPrioCount(9, 2)
+    call SetStartLocPrio(9, 0, 8, MAP_LOC_PRIO_HIGH)
+    call SetStartLocPrio(9, 1, 10, MAP_LOC_PRIO_HIGH)
+
+    call SetStartLocPrioCount(10, 2)
+    call SetStartLocPrio(10, 0, 9, MAP_LOC_PRIO_HIGH)
+    call SetStartLocPrio(10, 1, 11, MAP_LOC_PRIO_HIGH)
+
+    call SetStartLocPrioCount(11, 1)
+    call SetStartLocPrio(11, 0, 10, MAP_LOC_PRIO_HIGH)
+endfunction
+
+//***************************************************************************
+//*
+//*  Main Initialization
+//*
+//***************************************************************************
+
+//===========================================================================
+function main takes nothing returns nothing
+    call SetCameraBounds(- 7680.0 + GetCameraMargin(CAMERA_MARGIN_LEFT), - 8192.0 + GetCameraMargin(CAMERA_MARGIN_BOTTOM), 7680.0 - GetCameraMargin(CAMERA_MARGIN_RIGHT), 7168.0 - GetCameraMargin(CAMERA_MARGIN_TOP), - 7680.0 + GetCameraMargin(CAMERA_MARGIN_LEFT), 7168.0 - GetCameraMargin(CAMERA_MARGIN_TOP), 7680.0 - GetCameraMargin(CAMERA_MARGIN_RIGHT), - 8192.0 + GetCameraMargin(CAMERA_MARGIN_BOTTOM))
+    call SetDayNightModels("Environment\\DNC\\DNCLordaeron\\DNCLordaeronTerrain\\DNCLordaeronTerrain.mdl", "Environment\\DNC\\DNCLordaeron\\DNCLordaeronUnit\\DNCLordaeronUnit.mdl")
+    call SetWaterBaseColor(255, 255, 255, 255)
+    call NewSoundEnvironment("Default")
+    call SetAmbientDaySound("BarrensDay")
+    call SetAmbientNightSound("BarrensNight")
+    call SetMapMusic("Music", true, 0)
+    call CreateAllUnits()
+    call InitBlizzard()
+
+
+    call InitGlobals()
+
+endfunction
+
+//***************************************************************************
+//*
+//*  Map Configuration
+//*
+//***************************************************************************
+
+function config takes nothing returns nothing
+    call SetMapName("TRIGSTR_001")
+    call SetMapDescription("TRIGSTR_003")
+    call SetPlayers(12)
+    call SetTeams(12)
+    call SetGamePlacement(MAP_PLACEMENT_TEAMS_TOGETHER)
+
+    call DefineStartLocation(0, - 4608.0, 0.0)
+    call DefineStartLocation(1, 4608.0, 0.0)
+    call DefineStartLocation(2, - 6720.0, 1792.0)
+    call DefineStartLocation(3, - 6720.0, 896.0)
+    call DefineStartLocation(4, - 6720.0, 0.0)
+    call DefineStartLocation(5, - 6720.0, - 896.0)
+    call DefineStartLocation(6, - 6720.0, - 1792.0)
+    call DefineStartLocation(7, 6720.0, 1792.0)
+    call DefineStartLocation(8, 6720.0, 896.0)
+    call DefineStartLocation(9, 6720.0, 0.0)
+    call DefineStartLocation(10, 6720.0, - 896.0)
+    call DefineStartLocation(11, 6720.0, - 1792.0)
+
+    // Player setup
+    call InitCustomPlayerSlots()
+    call InitCustomTeams()
+    call InitAllyPriorities()
+endfunction
+
+
```

Comment:
I'm a bit confused by this change. Can you explain

Reply:
the war3map.j is actually a needed file to build your project (in case you reference stuff from it).
It should be in the git, while the other generated data should not.

Reply:
Ok, but didn't your PR remove this file?